### PR TITLE
VGA text buffer

### PR DESCRIPTION
**Pull request by rudyon**
Implements the VGA text buffer with a temporary text  function called `print_something()`

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
-
-    for (i, &byte) in HELLO.iter().enumerate() {
-        // Add ascii bit to buffer
-        // Then add color bit to buffer
-        unsafe {
-            *vga_buffer.offset(i as isize * 2) = byte;
-            *vga_buffer.offset(i as isize * 2 + 1) = light_cyan_color;
-        }
-    }
+    vga_buffer::print_something();
```
Comment (AuracleTech) [inline src/main.rs]: HELLO should be passed as variable here

File: src/vga_buffer.rs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                // not part of printable ASCII range
+                _ => self.write_byte(0xfe)
+            }
+        }
+    }
+
+    fn new_line(&mut self) {/* TODO */}
+}
+
+pub fn print_something() {
```
Comment (AuracleTech) [inline src/vga_buffer.rs]: Missing argument for text to print

File: src/vga_buffer.rs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    let mut writer = Writer {
+        column_position: 0,
+        color_code: ColorCode::new(Color::Yellow, Color::Black),
+        buffer: unsafe { &mut *(0xb8000 as *mut Buffer) }
+    };
+
+    writer.write_byte(b'H');
```
Comment (AuracleTech) [inline src/vga_buffer.rs]: Hardcoded values

**Review by AuracleTech — CHANGES_REQUESTED** [review]
Needs full implementation

**Review by rudyon — COMMENTED** [review]
Implements the VGA text buffer properly. Creates macros `print` and `println` and uses them to print cool ascii art.

**Pull request merged**